codebase2/codebase-sqlite/U/Codebase/Sqlite/Patch/TypeEdit.hs

```diff
@@ -17,7 +17,7 @@ type TypeEdit = TypeEdit' Db.TextId Db.ObjectId
 type HashTypeEdit = TypeEdit' Text ComponentHash
 
 data TypeEdit' t h = Replace (Reference' t h) | Deprecate
-  deriving (Eq, Ord, Show)
+  deriving (Eq, Functor, Ord, Show)
```

Comment:
Why add `Functor` on this? part of auto-deriving something else?

Reply:
Yup, if I don't add it then it errors out with:
```
U/Codebase/Sqlite/Patch/TypeEdit.hs:32:10: error: [GHC-39999]
     No instance for ‘Functor (TypeEdit' a)’
        arising from the head of a quantified constraint
        arising from the superclasses of an instance declaration
     In the instance declaration for ‘Bifunctor TypeEdit'’
   |
32 | instance Bifunctor TypeEdit' where
   |          ^^^^^^^^^^^^^^^^^^^
```